src/components/inputs/rich-text-input/dropdown.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import Downshift from 'downshift';
+import PropTypes from 'prop-types';
+import Tooltip from '../../tooltip';
+import AccessibleButton from '../../buttons/accessible-button';
+import {
+  getButtonStyles,
+  DropdownItem as StyledDropdownItem,
+  DropdownContainer,
+} from './dropdown.styles';
+
+// eslint-disable-next-line
+const Label = props => <span>{props.children}</span>;
+
+const Dropdown = props => {
+  const DropdownItem = props.components.DropdownItem || StyledDropdownItem;
+  const DropdownLabel = props.components.DropdownLabel || Label;
```

Comment:
`props.components` is not required if I read it correct. Either it has to or this needs to account/check for that.